### ISSUE TITLE
S1 - Accessibility issue; no alt text

### ISSUE THREAD

**Issue by jpinsent** [brharp/hjckrrh#34]
The social media icons that appear under 'Share this page' do not have alt text. They have descriptive title text that should instead be used for the alt text.

**Comment by Dmelady** [comment]
Please check the 'Follow us" icons as well. Thx

**Comment by brharp** [comment]
The social media icons are not images, so there is no alt text. We have added hidden text inside the link for screen readers, which should address the accessibility issue.
